lib/persisted-model.js

```diff
@@ -1072,6 +1066,54 @@ PersistedModel.enableChangeTracking = function() {
   }
 };
 
+function rectifyOnSave(ctx, next) {
+  if (ctx.instance) {
+    ctx.Model.rectifyChange(ctx.instance.getId(), reportErrorAndNext);
+  } else {
+    ctx.Model.rectifyAllChanges(reportErrorAndNext);
+  }
+
+  function reportErrorAndNext(err) {
+    if (err) {
+      console.error(
```

Comment:
The current implementation uses `console.error` too, see [lib/persisted-model.js#L1066-L1071](https://github.com/strongloop/loopback/blob/a82b33ec5c3cd043c53fed222ece2db67fc77c3c/lib/persisted-model.js#L1066-L1071). There are other places where PersistedModel prints to the console on error, I'd rather fix this properly in a standalone pull request - see #1126